### ISSUE TITLE
Path: quadratic Bezier continuation always returns zero coordinates (ignores parameters)

### ISSUE THREAD

**Issue by gnbl** [evtn/soda#3]
Something seems to be going wrong in https://github.com/evtn/soda/blob/lord/soda/paths.py#L76

```
from soda import Tag, Root, Path

commands = (
    Path.M(1, 2),
    Path.q(3, 4, 5, 6),
    Path.t( 7, 8),
    Path.Z()
)
p = Tag.path()(d=Path.build(*commands, sep=" "))

root = Root()(p)
print( root.render(pretty=True) )

```
Output

```
<svg
>
  <path
    d="M 1 2 q 3 4 5 6 t 0 0 Z"
  />
</svg>
```
Same with `T`:

`<svg><path d="M 1 2 q 3 4 5 6 T 0 0 Z"/></svg>`

**Comment by gnbl** [comment]
Appears to have too many parameters (and only returns the latter two):
`def q_shorthand(x1: int = 0, y1: int = 0, x: int = 0, y: int = 0, *, relative: bool = False):`

`Path.t(None,None, 7, 8),`
then returns 
`<svg><path d="M 1 2 q 3 4 5 6 t 7 8 Z"/></svg>`

**Comment by evtn** [comment]
Yes, I see, there are too many parameters in function, I will fix this.
Current workaround: `Path.t(x=7, y=8)`

Also, in your example `Root()(p)` could be shortened to `Root(p)`, or, as you don't use SVG namespaces, to `Tag.svg(p)`.

**Comment by gnbl** [comment]
I've removed the root attributes to keep it short.

Speaking of, I'm currently using

    ...
    svg = root.render(pretty=True)
    with open("svg.svg", "wt", encoding='utf-8') as f:
        f.write('<?xml version="1.0" encoding="UTF-8"?>')
        f.write(svg)

to build a valid file.
Do you think including an option to prepend the XML declaration or supply a custom preamble string makes sense?

**Comment by evtn** [comment]
I've fixed the original issue in 0.1.8 and created an issue for your request about XML declaration
